bool update
---
bool is_calibrating